fineract-provider/build.gradle

```diff
@@ -21,6 +21,9 @@ Run as:
 gradle clean bootRun'''
 
 project.ext.jerseyVersion = '1.19.4'
+ext["rest-assured.version"] = '4.3.0' 
```

Comment:
Also, I don't think this will be necessary since you can just define the dependency as is done https://github.com/apache/fineract/pull/662

Reply:
The reason why I did it this was that with a single line you can ensure that all the rest-assured libraries are brought in with the right version. I was first doing it through dependencies, but that seemed to require separately setting the version for every single rest-assured jar. That is of course much more error-prone: if you miss one, suddenly you are in classpath hell again. But if that's preferred I can of course set it jar-by-jar...

Reply:
Yeah, about that. spring dependency has an option to define a set of dependencies(dependencySet) for libraries which have thesame version number for different artifacts. There are samples of that in https://github.com/apache/fineract/pull/662. So you should be able to easily get the gist once you look that. With that you don't have to define dependencies jar-by-jar. Do you get what I mean here?

Reply:
Sorry - I thought the dependencySet only set the version for the JARs you've liste in the set, but not for the dependencies that those dependencies bring in (i.e. the transitive dependencies). 

But if it works for the non-listed transitive dependencies as well, then of course that will have the same result as this. I'll try it out and change if it works.

Reply:
Looks like it does not work for transitive dependencies - you need to explicitly list all the jars for which you want the version to apply. I've done this and listed the JARs we are currently using. But of course if we add some in the future then this list needs to be updated to avoid incompatible classes...